scripts/base/init-bare.zeek

```diff
@@ -4083,6 +4083,21 @@ type dns_edns_cookie: record {
 	server_cookie: string &default="";	##< Cookie from the server (0 bytes if missing, or 8 to 32 bytes).
 };
 
+## A DNS TKEY record.
+##
+## .. zeek:see:: dns_TKEY
+type dns_tkey: record {
+	query: string;		##< Query.
+	qtype: count;		##< Query type.
+	alg_name: string;	##< Algorithm name.
+	inception: time;	##< Time when signed.
+	expiration: time;	##< Time when the signed RR expires.
```

Comment:
Would be good to update to something else than the TSIG ones.

```suggestion
	inception: time;	##< Requested or provided start of validity interval for keying material.
	expiration: time;	##< Requested or provided end of validity interval for keying material.
```
https://datatracker.ietf.org/doc/html/rfc2930#section-2.4